apps/emqx_bridge_iotdb/include/emqx_bridge_iotdb.hrl

```diff
@@ -0,0 +1,13 @@
+%%--------------------------------------------------------------------
+%% Copyright (c) 2023 EMQ Technologies Co., Ltd. All Rights Reserved.
+%%--------------------------------------------------------------------
+
+-ifndef(EMQX_BRIDGE_IOTDB_HRL).
+-define(EMQX_BRIDGE_IOTDB_HRL, true).
+
+-define(VSN_1_0_X, 'v1.0.x').
+-define(VSN_0_13_X, 'v0.13.x').
+-define(VSN_1_0_X_BIN, <<"?VSN_1_0_X">>).
+-define(VSN_0_13_X_BIN, <<"?VSN_0_13_X">>).
```

Comment:
I think this doesn't work as intended :thinking: 

I made a quick local test:

```erlang
-define(YYY, 'test').
-define(XXX, <<"?YYY">>).

x() ->
    ?XXX.
```

Then:

```erlang
5> xxx:x().
<<"?YYY">>
```

Reply:
You're right 😓